### PR TITLE
Toolbar Buttons

### PR DESCRIPTION
**Pull request by azaleacolburn**
### Description
Add button that links to synthesis tutorials in the exporter toolbar.

### Testing Done
- [x] Click the button and go to the website

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1688)

### CODE REVIEW

**Comment by PepperLola** [comment]
When the addon starts, I get an error saying that 'UI.Helper' has no attribute 'check_solid_open' in `Synthesis.py:138`.

If I change it to `lambda *_: True` like what the rest of the buttons have, the new button shows up and opens the website properly.

Also, could you run the formatter?

File: exporter/SynthesisFusionAddin/Synthesis.py [hunk 1]
```diff
@@ -129,3 +130,14 @@ def register_ui() -> None:
     )
 
     gm.elements.append(apsButton)
+
+
+    websiteButton = HUI.HButton(
+        "Synthesis Website", 
+        work_panel,
+        Helper.check_solid_open,
```
Comment (BrandonPacewic) [inline exporter/SynthesisFusionAddin/Synthesis.py]: > When the addon starts, I get an error saying that 'UI.Helper' has no attribute 'check_solid_open' in `Synthesis.py:138`.

This was a merge conflict, left this to be changed in the HUI refactor.

```suggestion
        lambda *_: True,
```

**Review by BrandonPacewic — COMMENTED** [review]
@azaleacolburn I would recommend running the format script in [`exporter/SynthesisFusionAddin/Synthesis.py`](https://github.com/Autodesk/synthesis/blob/f467f95d3e62fe23b319135c2ebdd7a68dca643d/exporter/SynthesisFusionAddin/tools/format.py) if you are not already 🙃

**Review by PepperLola — APPROVED** [review]
Works now

**Review by BrandonPacewic — CHANGES_REQUESTED** [review]
Looks like there are still some left over formatting changes, for some reason some files are formatted to have max 80 character columns when it should be 120.

Reseting those files to their state on `dev` should do the trick.

**Review by BrandonPacewic — APPROVED** [review]
Just reverted the formatting changes that occurred 🙃

**Pull request merged**